tests/components/cambridge_audio/snapshots/test_select.ambr

```diff
@@ -103,6 +104,7 @@
         'bright',
         'dim',
         'off',
+        'none',
```

Comment:
What is the difference between off and none?

Does it even make sense for the user to set it to None?

Reply:
That's going to be the follow-up PR to this one. Some devices don't have displays at all and will return 'none' (enum type). This select won't be added at all if there isn't a display so this won't be an issue.

Reply:
See https://github.com/home-assistant/core/pull/130369

Reply:
Then I suggest closing this one in favor of #130369. Please update the other one to include the dependency bump.